packages/react-components/react-icons-compat/src/icon.ts

```diff
@@ -197,6 +197,7 @@ export function setIconOptions(options: Partial<IconOptions>): void {
 }
 
 let _missingIcons: string[] = [];
+// eslint-disable-next-line no-restricted-globals
```

Comment:
what's the approach here ? are we going to follow up with an issue assigned to owners to fix these violations or will you tackle this as a follow-up ? We better don't leave this hanging as that will kinda defeat the purpose of this great initiative.

Reply:
I'm not super happy with it but it's only in a few places for now so I can live with it.

I think the approach is this to [ignore TS types for this rule](https://github.com/microsoft/fluentui/issues/31286).  I don't see how it's valuable to exclude the types (in fact it feels counterproductive that the rules does that 🤷🤷‍♂️🤷‍♀️). I'll update all the instances where we're disabling for types to link to this follow up issue.

Then I'll work on implementing a lint rule that better suits our purpose.

Reply:
I've annotated all instances of this with 

`// TODO: exclude types from this lint rule: https://github.com/microsoft/fluentui/issues/31286`